### ISSUE TITLE
Pager template fails accessibility tests for Ellipsis ("...")

### ISSUE THREAD

**Issue by neyensp** [openfed/kiso#143]
@see https://www.drupal.org/project/drupal/issues/3348552

**Comment by neyensp** [comment]
The fix contain the post update hook to fix the translations for the 3 Belgium main languages.

```
drush updb
``` 

When we need to reimport the translations you can use the drush command

```
 # To import the kiso translations
 drush locale:import-all themes/contrib/kiso/translations
``` 

When a site instance has more languages then these 3. The customer need to add the translations by himself. This can be done to go to the next url '/admin/config/regional/translate' and search for the string "Ellipsis indicating non-visible pages"


[kiso-143-update-pager-ellipsis-aria-attribute.patch](https://github.com/user-attachments/files/22553921/kiso-143-update-pager-ellipsis-aria-attribute.patch)

**Comment by neyensp** [comment]
The "interface translation server pattern" was wrong therefore it doesn't find the local files.

Patch:

[kiso-143-update-pager-ellipsis-aria-attribute-3.patch](https://github.com/user-attachments/files/22763341/kiso-143-update-pager-ellipsis-aria-attribute-3.patch)

**Comment by ozmodiar** [comment]
Legacy patch:

[kiso-143-update-pager-ellipsis-aria-attribute-legacy.patch](https://github.com/user-attachments/files/22969271/kiso-143-update-pager-ellipsis-aria-attribute-legacy.patch)

**Comment by rutiolma** [comment]
Adding **aria-label** to a plain `<li>` generally does nothing, because list items without interactive or landmark roles are not typically focusable or announced individually by most of the screen readers. If you want to communicate “there are pages in between”, don’t rely on **aria-label** on an `<li>` element. Instead, provide **visually hidden text inside** the element:

```
<li class="ellipsis">
  <span aria-hidden="true">…</span>
  <span class="visually-hidden">Additional pages omitted</span>
</li>
```
